images/installer/Dockerfile.upi.ci

```diff
@@ -88,11 +88,12 @@ RUN mkdir /output && HOME=/output && \
     ibmcloud plugin install cloud-internet-services -f && \
     cp /usr/local/bin/ibmcloud /bin/ibmcloud && \
     rm -f $HOME/.curlrc && \
+    chmod -R g=u "$HOME/.bluemix/" && \
     ibmcloud version && \
     ibmcloud plugin list
 
 RUN chown -R 1000:1000 /output
```

Comment:
@MayXuQQ In https://github.com/openshift/installer/pull/6624/files, we changed the line to RUN chown -R, because we thought uid in container pod will be 1000, after that, we found the uid was not 1000, we have to use chmod -R g=u "$HOME/.bluemix/" to resolve it, so I suggested to change the line back to RUN chown 1000:1000 /output. Or else, do you mean `chmod -R g=u "$HOME/.bluemix/"` is not enough to resolve the permission issue, also need RUN chown -R 1000:1000 /output ?

Reply:
tested on local, `chown 1000:1000 /output` 

> drwx------. 1 root root 42 Nov 29 01:23 .bluemix

> drwxr-xr-x.   1 1000   1000     0 Nov 29 01:23 output

after add the  `RUN chown  -R 1000:1000 /output` after install the ibmcloud cli, get 

> drwx------. 1 1000 1000 42 Nov 29 01:23 .bluemix
> drwxr-xr-x.   1 1000   1000    16 Nov 29 01:23 output

Reply:
1. I do not think your local testing is totally the same as running in build farm cluster.
2. In the PR, per the result - `drwx------. 1 root root 42 Nov 29 01:23 .bluemix`, I did not see the change of `chmod -R g=u "$HOME/.bluemix/"`.

Reply:
You can see the output on https://github.com/openshift/installer/pull/6627#discussion_r1033084167, I got it from the environment you pointed me yesterday.  Its uid is `uid=1003320000`.